pkgs/top-level/aliases.nix

```diff
@@ -1030,6 +1030,7 @@ mapAliases ({
   pdf2htmlEx = throw "pdf2htmlEx has been removed from nixpkgs, as it was unmaintained"; # Added 2020-11-03
   pdfmod = throw "pdfmod has been removed"; # Added 2022-01-15
   pdfread = throw "pdfread has been remove because it is unmaintained for years and the sources are no longer available"; # Added 2021-07-22
+  peach = asouldocs; # Added 2022-08-28
```

Comment:
Hello. I need a feedback:

Do you have any preference between throwing a "package has been renamed" error and aliasing peach to the new asouldocs package? (like done in this file)

Reply:
Aliases that exist for some time get converted to throws and eventually removed, so this is correct.